src/System.Private.CoreLib/shared/System/Runtime/Intrinsics/Arm/AdvSimd.PlatformNotSupported.cs

```diff
@@ -549,6 +549,132 @@ public new abstract class Arm64 : ArmBase.Arm64
         /// </summary>
         public static Vector128<ulong> AndNot(Vector128<ulong> left, Vector128<ulong> right) { throw new PlatformNotSupportedException(); }
 
+        /// <summary>
+        /// int8x8_t vcls_s8 (int8x8_t a)
+        ///   A32: VCLS Dd, Dm
+        ///   A64: CLS Vd, Vn
+        /// </summary>
+        public static Vector64<sbyte> LeadingSignCount(Vector64<sbyte> value) { throw new PlatformNotSupportedException(); }
```

Comment:
We should probably exposed overloads for `byte`, `ushort`, and `uint` (And same for the scalar versions)? 

The operation only counts the number of bits following the most significant bit that are the same as the most significant bit (exclusing the most significant bit itself). This logically applies to unsigned types as well.

Reply:
I can add those. The reason why I don't have them here because I followed the arm Intrinsics Reference and they have only signed argument versions in there.

 Any idea how `LeadingSignCount` intrinsic can be actually used?

Reply:
> Any idea how LeadingSignCount intrinsic can be actually used?

I'm not familiar with any particular bit manipulation tricks that rely on this particular behavior.

Reply:
I believe that it can be used to compute stuff like FitsIn16, FitsIn8 
It is a way to determine how many bits are need to represent the value in a signed format.

Reply:
> I can add those. The reason why I don't have them here because I followed the arm Intrinsics Reference and they have only signed argument versions in there.

Yeah I think that's an omission, I've sent a patch to correct that :)